homeassistant/components/blink/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "blink",
   "name": "Blink",
   "documentation": "https://www.home-assistant.io/integrations/blink",
-  "requirements": ["blinkpy==0.19.2"],
+  "requirements": ["blinkpy==0.20.0.rc0"],
```

Comment:
Why is it RC? Unless we are fixing a completely broken integration and have no other way, we do not allow the use of dev/alpha/beta/release candidates.

Reply:
Understood.

The original Blink integration is not broken.  But the `0.19.x` versions do not work _at all_ for those of use who don't use Blink's cloud storage.  In that sense, it is broken/unsupported.

@fronzbot :  Are you ready to release `blinkpy` updates as `v0.20.0`?

Reply:
This PR is now using the full release version `0.20.0`.  It is ready for another review @frenck.  Thanks!